### PR TITLE
Update Terraform to create a new client for AB#14646

### PR DESCRIPTION
**Pull request by sslaws**
# Fixes or Implements [AB#14646](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/14646)

## Description
Adds a new client to Keycloak to be used for the Web Queue Ticket system
Ran the Terraform format command which formatted idp_phsa

## Testing

- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [X] Not Required

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

**Review by ytqsl — APPROVED** [review]

File: Tools/KeyCloak/Terraform/idp_phsa.tf [hunk 1]
```diff
@@ -1,21 +1,21 @@
 resource "keycloak_oidc_identity_provider" "phsa" {
-  realm                 = data.keycloak_realm.hg_realm.id
-  alias                 = "phsaazure"
-  display_name          = "PHSA Azure"
-  enabled               = true
-  store_token           = false
-  trust_email           = true
+  realm                         = data.keycloak_realm.hg_realm.id
```
Comment (BrianMaki) [inline Tools/KeyCloak/Terraform/idp_phsa.tf:2]: Lines 2 to 7, is there supposed to be this added space?

File: Tools/KeyCloak/Terraform/idp_phsa.tf [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-  sync_mode             = "FORCE"
-  authorization_url     = "${var.keycloak_idp_phsa.base_url}${var.keycloak_idp_phsa.auth_path}"
-  token_url             = "${var.keycloak_idp_phsa.base_url}${var.keycloak_idp_phsa.token_path}"
-  backchannel_supported = false
-  client_id             = var.keycloak_idp_phsa.client_id
-  client_secret         = var.keycloak_idp_phsa.client_secret
-  default_scopes        = "openid profile email"
-  validate_signature    = true
-  jwks_url              = "${var.keycloak_idp_phsa.base_url}${var.keycloak_idp_phsa.jwks_path}"
+  hide_on_login_page            = false
```
Comment (BrianMaki) [inline Tools/KeyCloak/Terraform/idp_phsa.tf:9]: Lines 9 to 18, is there supposed to be this added space?

**Review by BrianMaki — COMMENTED** [review]
Left question regarding spacing.

**Review by ytqsl — APPROVED** [review]

File: Tools/KeyCloak/Terraform/idp_phsa.tf (hunk 2 above)
Comment (sslaws) [reply Tools/KeyCloak/Terraform/idp_phsa.tf:9]: I'm not sure what is being displayed here but in VS Code I don't see any of those spaces.  Terraform has its only formatter which I run on a regular basis.

**Review by BrianMaki — APPROVED** [review]

**Pull request merged**